.vitepress/theme/components/BannerComponent.vue

```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
+import { useElementSize } from '@vueuse/core';
+import { onBeforeMount, onMounted, ref, watchEffect } from 'vue';
+
+const el = ref<HTMLElement>();
+const { height } = useElementSize(el);
+
+watchEffect(() => {
+  if (height.value) {
+    document.documentElement.style.setProperty(
+      '--vp-layout-top-height',
+      `${height.value + 16}px`
+    );
+  }
+});
+
+const dismiss = () => {
+  localStorage.setItem(
+    'dismissBanner',
+    (Date.now() + 8.64e7 * 1).toString() // current time + 1 day
```

Comment:
This doesnt seem to be working.

why `* 1`?

`8.64e7` is not clear, just keep it simple and use something like: `24 * 60 * 60 * 1000`